Cargo.toml

```diff
@@ -16,9 +16,9 @@ strum = "0.17"
 strum_macros = "0.17"
 serde = "1"
 serde_derive = "1"
-wasm-bindgen = "0.2"
+wasm-bindgen = "0.2.58"
 web_logger = "0.2"
-yew = "0.12"
+yew = { version = "0.13", features = ["web_sys"] }
 
 # The `console_error_panic_hook` crate provides better debugging of panics by
```

Comment:
Could you also please remove this whole section for `console_error_panic_hook` (including the dependency)? It is turned on by default now in Yew

Reply:
sure, done